contract/src/main/java/io/neow3j/contract/NonFungibleToken.java

```diff
@@ -43,6 +48,43 @@ public NonFungibleToken(Hash160 scriptHash, Neow3j neow) {
         super(scriptHash, neow);
     }
 
+    /**
+     * Gets the total amount of NFTs owned by the {@code owner}.
+     * <p>
+     * The balance is not cached locally. Every time this method is called requests are send to
+     * the Neo node.
+     *
+     * @param owner the script hash of the account to fetch the balance for.
+     * @return the token balance of the given account.
+     * @throws IOException                   if there was a problem fetching information from
+     *                                       the Neo node.
+     * @throws UnexpectedReturnTypeException if the contract invocation did not return something
+     *                                       interpretable as a number.
+     */
+    public BigInteger balanceOf(Hash160 owner)
+            throws IOException {
+        return callFuncReturningInt(BALANCE_OF, hash160(owner));
+    }
+
+    /**
+     * Gets the token ids of the tokens that are owned by the {@code owner}.
+     * <p>
+     * Consider that for this RPC the returned list may be limited in size and not reveal all
+     * entries that exist on the contract.
+     *
+     * @param owner the owner of the tokens.
+     * @return a list of token ids that are owned by the specified owner.
+     * @throws IOException if there was a problem fetching information from the Neo node.
+     */
+    public List<byte[]> tokensOf(Hash160 owner) throws IOException {
+        return callFunctionReturningIterator(TOKENS_OF, hash160(owner))
+                .stream()
+                .map(StackItem::getByteArray)
+                .collect(Collectors.toList());
+    }
+
+    // Non-divisible NFT methods
+
     /**
      * Creates a transaction script to transfer a non-fungible token and initializes a
      * {@link TransactionBuilder} based on this script.
```

Comment:
Add the following comment here too, as you have added it to the other overload of this method.
"This method is intended to be used for non-divisible NFTs only."